<?xml version="1.0" encoding="UTF-8"?>

<launch>

	<arg name="mav_name" default="quad"/>
	<arg name="id" default="3"/>
	<arg name="quad_ns"  default="$(arg mav_name)$(arg id)"/>
	<arg name="fixed_height" default="1.5"/>
	<arg name="max_vel" default="0.5"/>
	
	<group ns="$(arg mav_name)$(arg id)">
		<node pkg="mslquad" type="default_controller" name="default_controller" output="screen">
			<param name="quad_ns" value="$(arg quad_ns)"/>
			<param name="fixed_height" value="$(arg fixed_height)" />
			<param name="max_vel" value="$(arg max_vel)" />
		</node>

	</group>
</launch>